app/uk/gov/hmrc/tariffclassificationfrontend/service/QueuesService.scala

```diff
@@ -25,6 +25,10 @@ class QueuesService {
 
   def getAll: Seq[Queue] = Seq(gateway, act, cap, cars, elm)
 
+  def queueNameForId(id: String): String = {
```

Comment:
You could have just got the full Queue and used the existing methods

Reply:
Then do `queue.name` in the view

Reply:
OK. The `find` will return an `Option[Queue]` and the view can deal with that